### PR TITLE
Fix git queries for OPENJDK_TAG

### PR DESCRIPTION
**Pull request by keithc-ca**
Only match openjdk tags whose names start with 'jdk8u'.

Adjust whitespace as requested for #35.

### CODE REVIEW

**Comment by pshipton** [comment]
jdk/make/closed/autoconf/custom-hook.m4 also needs to be fixed the same way.

The description of the commit could be more explicit, i.e. "Fix openjdk tag query" or "Fix OPENJDK_TAG query"

File: common/autoconf/configure [hunk 1]
```diff
@@ -123,10 +123,10 @@ fi
 ###
 # Add default OpenJ9 closed extensions
 full_script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-OPENJDK_TAG=`git -C $full_script_dir/../.. describe --abbrev=0 --tags`
+OPENJDK_TAG=`git -C $full_script_dir/../.. describe --abbrev=0 --tags --match "jdk8u*"`
```
Comment (pshipton) [inline common/autoconf/configure]: JDK9 contains `--match "jdk-9*" "${OPENJDK_SHA}"`. Does the SHA also need to be specified here?

Comment (keithc-ca) [reply common/autoconf/configure]: I don't think it is needed; git assumes 'HEAD', but I'll add it for consistency.

**Comment by keithc-ca** [comment]
custom-hook.m4 (and files derived via autogen.sh) updated along with commit description.

**Comment by pshipton** [comment]
This repo doesn't support pull request testing yet.

**Pull request merged**